examples/nodejs/src/index.ts

```diff
@@ -0,0 +1,4 @@
+import { Cohort, getPorterUri } from '@nucypher/shared';
```

Comment:
This example was trimmed down in the refactor. In the case of all examples, I will have to update them to use `taco` and `pre` instead of `shared`, so I expect at least some changes here.